### PR TITLE
Add support for Gizz Tapes

### PR DESCRIPTION
**Pull request by netclectic**
**Describe the changes you made**
 - added connector for Gizz Tapes: `tapes-kglw.ts` - https://tapes.kglw.net/
 - added connector details to core/connectors.ts

### CODE REVIEW

File: src/connectors/tapes-kglw.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Connector.albumSelector =
+	'#site-header div#now-playing > span#now-playing-show-date';
+
+Connector.currentTimeSelector =
+	'#site-header div#now-playing > span.now-playing-elapsed';
+
+Connector.durationSelector =
+	'#site-header div#now-playing > span.now-playing-duration hidden-sm';
+
+Connector.playButtonSelector = '#play-btn';
```
Comment (inverse) [inline src/connectors/tapes-kglw.ts:21]: I don't think this is needed as you're overriding the isPlaying function below.

**Review by inverse — COMMENTED** [review]
Seems reasonable aside from the one comment!

File: src/connectors/tapes-kglw.ts (hunk 1 above)
Comment (netclectic) [reply src/connectors/tapes-kglw.ts]: Good to know, updated.

**Review by inverse — APPROVED** [review]
Thanks for the contribution!

File: src/connectors/tapes-kglw.ts (hunk 1 above)
Comment (inverse) [reply src/connectors/tapes-kglw.ts]: The duration one could have been left in I think but perhaps could be updated to 


```
Connector.durationSelector =
	'#now-playing > span.now-playing-duration';
```

**Pull request merged**